### ISSUE TITLE
about code

### ISSUE THREAD

**Issue by njnj0314** [YilingQiao/intrinsicSym#1]
Hello, when I run train.py, the following occurs:
python train.py But the code seems to be stuck, there is no new operating condition update

————————————————————————————————————————

sca
../data/annotated/test/scape_mesh071_anno.mat
(96,)
starting training loop...
2021-12-06 18:21:00.291351: W tensorflow/core/framework/allocator.cc:124] Allocation of 294912000 exceeds 10% of system memory.
2021-12-06 18:21:01.143025: W tensorflow/core/framework/allocator.cc:124] Allocation of 294912000 exceeds 10% of system memory.
2021-12-06 18:21:01.233711: W tensorflow/core/framework/allocator.cc:124] Allocation of 294912000 exceeds 10% of system memory.
2021-12-06 18:21:01.600829: W tensorflow/core/framework/allocator.cc:124] Allocation of 294912000 exceeds 10% of system memory.
2021-12-06 18:21:02.469362: W tensorflow/core/framework/allocator.cc:124] Allocation of 294912000 exceeds 10% of system memory.
test-epoch 0: loss = 0.4989, acc = 0.7812

**Comment by YilingQiao** [comment]
Seems that you do not have enough memory. Do you run it on CPU or GPU? Can you check how much your memory or GPU memory is?

**Comment by njnj0314** [comment]
I run it on GPU. 8GB.
